__tests__/app.test.js

```diff
@@ -179,6 +190,53 @@ describe("/api/articles/:article_id", () => {
   })
 })
 
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+      test("Status 200 - returns an article object", () => {
+          return request(app)
+          .get('/api/articles/3/comments')
+          .expect(200)
+          .then(({body}) => {
+              expect(body).toBeInstanceOf(Object);
+              expect(body).toHaveProperty('comments');
+              expect(Array.isArray(body.comments)).toBe(true);
+              expect(body.comments[0]).toEqual(
+                expect.objectContaining({
+                  comment_id: 10,
+                  votes: 0,
+                  created_at: "2020-06-20T07:24:00.000Z",
+                  author: "icellusedkars",
+                  body: "git push origin master"
+                }))
+              body.comments.forEach(comment => {expect(comment).
+                toEqual(expect.objectContaining({
+                  comment_id: expect.any(Number),
+                  votes: expect.any(Number),
+                  created_at: expect.any(String),
+                  author: expect.any(String),
+                  body: expect.any(String),
+                }))})  
```

Comment:
Would definitely also check the comments array has the right length that you were expecting